### PR TITLE
SEO-175653-Xamarin-Pages-With-Redirects

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

Worked on the links with redirects.

Regards,
Beryl.

### CODE REVIEW

File: Xamarin/Busy-Indicator/Getting-Started.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ This section explains you the steps to configure a [`SfBusyIndicator`](https://h
 
 ## Assembly deployment
 
-After installing [Essential Studio for Xamarin](https://www.syncfusion.com/downloads/xamarin/confirm), you can find all the required assemblies in the installation folders, {Syncfusion Essential Studio Installed location} \Essential Studio\\{Version #}\Xamarin\lib.
+After installing [Essential Studio for Xamarin](https://www.syncfusion.com/downloads), you can find all the required assemblies in the installation folders, {Syncfusion Essential Studio Installed location} \Essential Studio\\{Version #}\Xamarin\lib.
```
Comment (Yvone-Atieno) [inline Xamarin/Busy-Indicator/Getting-Started.md:18]: revert this change

File: Xamarin/Calendar/Getting-Started.md [hunk 2]
```diff
@@ -60,9 +60,9 @@ Location: {Installed location}/{version}/Xamarin/lib
...
+N> To know more about obtaining our components, refer to these links for [Mac](https://help.syncfusion.com/xamarin/installation/mac-installer/how-to-download) and [Windows](https://help.syncfusion.com/xamarin/installation/web-installer/how-to-download).
 
-I> Starting with v16.2.0.x, if you reference Syncfusion assemblies from the trial setup or from the NuGet feed, you also have to include a license key in your projects. Please refer to [Syncfusion license key](https://help.syncfusion.com/common/essential-studio/licensing/license-key/) to know about registering Syncfusion license key in your Xamarin application to use our components.
+I> Starting with v16.2.0.x, if you reference Syncfusion assemblies from the trial setup or from the NuGet feed, you also have to include a license key in your projects. Please refer to [Syncfusion license key](https://help.syncfusion.com/common/essential-studio/licensing/how-to-generate) to know about registering Syncfusion license key in your Xamarin application to use our components.
```
Comment (Yvone-Atieno) [inline Xamarin/Calendar/Getting-Started.md:65]: Update the correct link

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: Xamarin/Busy-Indicator/Getting-Started.md (hunk 1 above)
Comment (berylouma) [reply Xamarin/Busy-Indicator/Getting-Started.md:18]: Updated the correct link

File: Xamarin/Calendar/Getting-Started.md (hunk 2 above)
Comment (berylouma) [reply Xamarin/Calendar/Getting-Started.md]: Updated

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: Xamarin/Calendar/Getting-Started.md [hunk 3]
```diff
@@ -60,9 +60,9 @@ Location: {Installed location}/{version}/Xamarin/lib
...
+N> To know more about obtaining our components, refer to these links for [Mac](https://help.syncfusion.com/xamarin/installation/mac-installer/how-to-download) and [Windows](https://help.syncfusion.com/xamarin/installation/web-installer/how-to-download).
 
-I> Starting with v16.2.0.x, if you reference Syncfusion assemblies from the trial setup or from the NuGet feed, you also have to include a license key in your projects. Please refer to [Syncfusion license key](https://help.syncfusion.com/common/essential-studio/licensing/license-key/) to know about registering Syncfusion license key in your Xamarin application to use our components.
+I> Starting with v16.2.0.x, if you reference Syncfusion assemblies from the trial setup or from the NuGet feed, you also have to include a license key in your projects. Please refer to [Syncfusion license key](https://help.syncfusion.com/common/essential-studio/licensing/how-to-register-in-an-application#where-to-register-the-syncfusion-license-key) to know about registering Syncfusion license key in your Xamarin application to use our components.
```
[the remark this answers is not in the document]
Comment (berylouma) [reply Xamarin/Calendar/Getting-Started.md]: Updated

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Fine.

**Pull request merged**